### DISCUSSION TITLE
20240620 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#99] [Meetings]
- Time:  9:30 in Canary Islands (WET (UTC+0)).
- Time: 10:30 in Vaduz (CET (UTC+1)).

Location:
<https://meeting.blockfinance-eco.li/NautilusCyberneeringWeeklyDevMeeting>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- Cameron Garnham(@da2ce7)  - x
- Jose Celano(@josecelano) - x
- Constantin Bosse(@cgbosse) - x
- Mario Melchor(@mario-nt) - x
- Graeme Byrne(@grmbyrn) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
In short, my main focus has been on version 2 of the configuration TOML files for the Tracker and the Index (with breaking changes).

### Torrust Tracker

Issues:

- https://github.com/torrust/torrust-tracker/issues/893
- https://github.com/torrust/torrust-tracker/issues/878

PRs:

- [Update dependencies](https://github.com/torrust/torrust-tracker/pull/906)

### Torrust Index

Issues:

- https://github.com/torrust/torrust-tracker/issues/878

PRs:

- [Update dependencies](https://github.com/torrust/torrust-index/pull/639)

PR review:

- https://github.com/torrust/torrust-index/pull/624

### Torrust Index GUI

Issues:

- https://github.com/torrust/torrust-index-gui/issues/573
- https://github.com/torrust/torrust-index-gui/issues/574

PR review:

- https://github.com/torrust/torrust-index-gui/pull/573

### Torrust Demo

Issues:

- https://github.com/torrust/torrust-demo/issues/6

**Comment by mario-nt** [comment]
### Index

- https://github.com/torrust/torrust-index/issues/615

> Handlers that require an admin user
> New service and authorization logic for public handlers 

### Reviews

- https://github.com/torrust/torrust-index-gui/issues/527#issuecomment-2163082970

### Training/Research

- Rust language programming book

### Other

-  Fix index not finding database file
